doc/source/cql/types.rst

```diff
@@ -400,20 +398,46 @@ Further, lists support:
 
 Lastly, as for :ref:`maps <maps>`, TTLs when used only apply to the newly inserted values.
 
+.. _tuples:
+
+Tuples
+^^^^^^
+
+CQL also support tuples and tuple types (where the elements can be of different types). Tuple types and tuple literals
+are defined by:
+
+.. productionlist::
+   tuple_type: TUPLE '<' `cql_type` ( ',' `cql_type` )* '>'
+   tuple_literal: '(' `term` ( ',' `term` )* ')'
+
+and can be used thusly::
+
+    CREATE TABLE durations (
+        event text,
+        duration tuple<int, text>,
+    )
+
+    INSERT INTO durations (event, duration) VALUES ('ev1', (3, 'hours'));
+
+Unlike other "composed" types (collections and UDT), a tuple is always :ref:`frozen <frozen>` (without the need of the
+`frozen` keyword) and it is not possible to update only some elements of a tuple (without updating the whole tuple).
+Also, a tuple literal should always have the same number of value than declared in the type it is a tuple of (some of
+those values can be null but they need to be explicitly declared as so).
+
 .. _udts:
 
 User-Defined Types
 ^^^^^^^^^^^^^^^^^^
 
-CQL support the definition of user-defined types (UDT for short). Such a type can be created, modified and removed using
-the :token:`create_type_statement`, :token:`alter_type_statement` and :token:`drop_type_statement` described below. But
+CQL support the definition of user-defined types (UDT for short), which is basically a tuple on steroids - it's
+literally an extension of the class that represents a tuple. Such a type can be created, modified and removed using the
```

Comment:
It's better to keep implementation details out of these docs, I think - I'm not sure it's necessary to specify that a UDT is a form of tuple.  Much as I like it, I think docs should also perhaps use less interesting language? (so "on steroids" might better be avoided, but just my 2c)

Reply:
Hey, @belliottsmith! All other comments have been addressed. Thanks a lot!
About this one... How do you feel about this?
```
CQL support the definition of user-defined types (UDT for short), which is basically an improved tuple.
```

Reply:
Hey, @blerer! Thanks for the review!
Do you have any suggestions on how should I write this? I thought the aforementioned suggestion was fine, but I'm not sure how to improve it. :confused:

Reply:
The important part for me was to not mention the link with the `tuple` because UDTs and tuple are different in a lot of aspects.
What I had in mind was something like:
`A User Defined Type (UDT) is a set data fields where each field is named and typed. UDTs allow to store related information together within one colum. UDTs can be created, modified and removed using ...`

Reply:
Done, just as you suggested, @blerer! :upside_down_face: